app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -1089,7 +1093,7 @@ class AdvancedFilter extends React.Component {
             )}
           </Col>
           <Col className="py-0" md="auto">
-            {filterOption && (filterOption.tooltip || numberOption === 'between' || relativeOption === 'custom') && (
+            {filterOption && (filterOption.tooltip || numberOption === 'between' || relativeOption === 'custom' || additionalFilterOption === 'Multiple') && (
```

Comment:
The tooltip handling here is getting to be a mess and hard to follow. This is more of a comment for awareness rather than a request - but I think we may need to make a ticket to make this functionality more usable in this component.

Reply:
I know - the problem with this there are three different cases that we are trying to handle that are all pretty different..

1. Tooltips that are filter field specific (defined as tooltip on the object)
2. Tooltips that are filter type specific (i.e. relative dates)
3. Tooltips that are filter field specific but should only be shown if other filter values meet a certain set of requirements

To be honest I have no idea how to best do this.  Would be good to make a ticket though

Reply:
https://tracker.codev.mitre.org/browse/SARAALERT-1318